### PR TITLE
Use one channel for events instead of three

### PR DESCRIPTION
**Pull request by nimrod-up9**

### CODE REVIEW

File: cli/cmd/tapRunner.go [hunk 1]
```diff
@@ -559,30 +559,24 @@ func waitUntilNamespaceDeleted(ctx context.Context, cancel context.CancelFunc, k
...
-				continue
-			}
-
-			logger.Log.Infof("%s removed", kubernetes.ApiServerPodName)
-			cancel()
-			return
-		case wEvent, ok := <-modified:
-			if !ok {
-				modified = nil
+			if wEvent.Type == kubernetes.EventAdded {
```
Comment (RoyUP9) [inline cli/cmd/tapRunner.go:573]: maybe change to switch case like in mizuTapperSyncer

File: cli/cmd/tapRunner.go [hunk 2]
```diff
@@ -654,61 +648,40 @@ func watchApiServerPod(ctx context.Context, kubernetesProvider *kubernetes.Provi
...
 				cancel()
 				continue
 			}
 
-
-			logger.Log.Debugf("Tapper is removed [%s]", removedPod.Name)
-		case wEvent, ok := <-modified:
-			if !ok {
-				modified = nil
+			if wEvent.Type == kubernetes.EventAdded {
```
Comment (RoyUP9) [inline cli/cmd/tapRunner.go:668]: maybe change to switch case like in mizuTapperSyncer

File: cli/cmd/tapRunner.go [hunk 3]
```diff
@@ -559,30 +559,25 @@ func waitUntilNamespaceDeleted(ctx context.Context, cancel context.CancelFunc, k
...
-				continue
-			}
-
-			logger.Log.Infof("%s removed", kubernetes.ApiServerPodName)
-			cancel()
-			return
-		case wEvent, ok := <-modified:
-			if !ok {
-				modified = nil
+			switch {
```
Comment (RoyUP9) [inline cli/cmd/tapRunner.go:573]: you can change it to switch wEvent.Type

File: cli/cmd/tapRunner.go [hunk 4]
```diff
@@ -654,70 +649,51 @@ func watchApiServerPod(ctx context.Context, kubernetesProvider *kubernetes.Provi
...
 				logger.Log.Errorf(uiUtils.Error, err)
 				cancel()
 				continue
 			}
 
-			logger.Log.Debugf("Tapper is created [%s]", addedPod.Name)
-		case wEvent, ok := <-removed:
-			if !ok {
-				removed = nil
+			switch {
```
Comment (RoyUP9) [inline cli/cmd/tapRunner.go:669]: switch wEvent.Type

File: cli/cmd/tapRunner.go (hunk 1 above)
Comment (nimrod-up9) [reply cli/cmd/tapRunner.go]: changed

File: cli/cmd/tapRunner.go (hunk 2 above)
Comment (nimrod-up9) [reply cli/cmd/tapRunner.go]: changed

**Review by RoyUP9 — APPROVED** [review]

**Pull request merged**